lib/src/video_player/method_channel_video_player.dart

```diff
@@ -493,6 +482,11 @@ class MethodChannelVideoPlayer extends VideoPlayerPlatform {
             eventType: VideoEventType.failedToPlayToEndTime,
             key: key,
           );
+        case 'isBackToAppButtonPressed':
```

Comment:
`is` prefix is usually used as "function that returns bool" or "bool variable name".
How about renaming this and the same names to `tapBackToAppButton`? (or `pressedBackToAppButton`, etc..)